### PR TITLE
Italian translations

### PR DESCRIPTION
**Pull request by Guglieee**

### CODE REVIEW

**Review by Guglieee — COMMENTED** [review]
Fixed some things

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 1]
```diff
@@ -435,6 +469,7 @@
             put(Language.ET, "URLi allalaadimine ebaõnnestus");
             put(Language.ZH, "自URL下載失敗");
             put(Language.PL, "Nie udało się pobrać adresu URL");
+            put(Language.IT, "Failed to download URL");
```
Comment (ANutley) [inline src/main/java/github/scarsz/discordsrv/util/LangUtil.java:472]: Need to translate this line

File: src/main/resources/config/it.yml [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+# DiscordChatChannelBroadcastDiscordMessagesToConsole: Se stampare o meno i messaggi di Discord elaborati alla console
+# DiscordChatChannelRequireLinkedAccount: Se richiedere o meno ai giocatori hanno il loro account di Discord collegato al loro minecraft per avere i loro messaggi di Discord inviati a Minecraft
+# DiscordChatChannelBlockBots: whether or not bots should be blocked from Discord -> MC chat
+# DiscordChatChannelBlockedIds: ids of discord users (or bots) that should not have their messages processed & sent to MC
+# DiscordChatChannelBlockedRolesAsWhitelist: if the following list should be treated as a whitelist (true) or a blacklist (false)
+# DiscordChatChannelBlockedRolesIds: ids of discord roles that should not have their messages processed & sent to MC
+# DiscordChatChannelRolesSelectionAsWhitelist: if the following list should be treated as a whitelist (true) or a blacklist (false)
+# DiscordChatChannelRolesSelection: list of roles that should be filtered from all of a user's roles
+# DiscordChatChannelRoleAliases: list of role aliases (alternate names for roles to use in Minecraft messages)
```
Comment (ANutley) [inline src/main/resources/config/it.yml:159]: Needs to be translated

**Review by ANutley — CHANGES_REQUESTED** [review]

File: src/main/resources/config/it.yml (hunk 2 above)
Comment (Guglieee) [reply src/main/resources/config/it.yml:159]: Fixed

**Review by Guglieee — COMMENTED** [review]
Fixed All

**Comment by Guglieee** [comment]
I think I have translated everything, if while checking you find something untranslated just tell me and I will translate immediately

**Review by Vankka — CHANGES_REQUESTED** [review]
A few things I noticed listed below

**Comment by Guglieee** [comment]
I'm going to work this afternoon

File: src/main/resources/messages/it.yml [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+#                    esempio: Notch
+# %channelname%:    nome del canale da cui proviene il messaggio
+#                    esempio: server-chat
+# %reply%:          Il messaggio visualizzato quando il messaggio è una risposta a un altro messaggio.
+#                    Il formato del messaggio può essere configurato con DiscordToMinecraftMessageReplyFormat,
+#                    Questo sarà vuoto se il messaggio non è una risposta a un altro messaggio
+#
+# DiscordToMinecraftAllRolesSeparator: il separatore utilizzato tra i ruoli in %allroles%
+#
+# DiscordToMinecraftMessageReplyFormat: ilFormatoPerIlMessaggioVisualizzatoPerIndicareCheIlMessaggioÈUnaRisposta
```
[the remark this answers is not in the document]
Comment (Guglieee) [reply src/main/resources/messages/it.yml:37]: Solved, my mistake

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 4]
```diff
@@ -80,6 +81,21 @@
...
+                    "\n" +
+                    "Istruzioni per l'aggiornamento a ASM 5:\n" +
+                    "1. Navigare verso {specialsourcefolder} Cartella del server\n" +
+                    "2. Elimina il file JAR SpecialSource-1.7-Snapshot.jar\n" +
+                    "3. Scarica SpecialSource v1.7.4 da https://repo1.maven.org/maven2/net/md-5/SpecialSource/1.7.4/SpecialSource-1.7.4.jar\n" +
+                    "4. Copia il file JAR in {specialsourcefolder} cartella del server che hai navigato in precedenza\n" +
+                    "5. Rinominare il file JAR che hai appena copiato in SpecialSource-1.7-snapshot.jar.jar\n" +
+                    "6. Riavvia il server\n" +
+                    "\n" +
+                    "\n");
```
[the remark this answers is not in the document]
Comment (Guglieee) [reply src/main/java/github/scarsz/discordsrv/util/LangUtil.java:98]: Solved

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 5]
```diff
@@ -234,6 +250,15 @@
...
                     "\n");
+            put(Language.IT,
+                    "\n" +
+                    "\n" +
+                    "RisperazionePlugins è disabilitato, questa opzione è solo a scopo di test\n" +
+                    "e non dovrebbe mai essere disabilitato sui server di produzione.\n" +
+                    "Disabilitare l'opzione causerà inoltre inoltrato messaggi annullati a Discord\n" +
+                    "incluso ma non limitato a messaggi privati o messaggi di chat del personale senza /commands\n" +
+                    "\n" +
+                    "\n");
```
[the remark this answers is not in the document]
Comment (Guglieee) [reply src/main/java/github/scarsz/discordsrv/util/LangUtil.java:261]: OK

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java [hunk 6]
```diff
@@ -555,6 +599,7 @@
             put(Language.ET, "Ühendatud kontod salvestati {ms}ms jooksul");
             put(Language.ZH, "已儲存已連結帳號，耗時{ms}ms");
             put(Language.PL, "Zapisane połączone konta w {ms}ms");
+            put(Language.IT, "Account collegati salvati in {ms} ms");
```
[the remark this answers is not in the document]
Comment (Guglieee) [reply src/main/java/github/scarsz/discordsrv/util/LangUtil.java:602]: My bad

**Review by Guglieee — COMMENTED** [review]
I've looked at it now too and there are no mistakes (at least I think)

File: src/main/java/github/scarsz/discordsrv/util/LangUtil.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (Vankka) [reply src/main/java/github/scarsz/discordsrv/util/LangUtil.java]: > Please move this below the Polish version

The position still needs to be adjusted

**Comment by Guglieee** [comment]
@Vankka NOW I have moved it
😄

**Comment by Guglieee** [comment]
So?

File: src/main/resources/config/it.yml [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+# DiscordConsoleChannelLogRefreshRateInSeconds: ogni quante righe nella console, le invia su discord
+# DiscordConsoleChannelUsageLog:
+#    %date%:  current date
+#              example: 2017-01-01
+#    PlaceholderAPI placeholders are supported
+# DiscordConsoleChannelBlacklistActsAsWhitelist: Se l'elenco dei comandi Blacklist o no agisce come whitelist invece della lista nera
+# DiscordConsoleChannelBlacklistedCommands: Le frasi avvolte in virgolette che gli utenti non dovrebbero essere in grado di inviare come comandi alla console
+# DiscordConsoleChannelFilters: I filtri Regex devono essere applicati alle linee della console inviate alla discordia, se il risultato è vuoto il messaggio non verrà inviato affatto
+# DiscordConsoleChannelLevels: Livelli da inviare al canale della console tramite Antepender
+# DiscordConsoleChannelUseCodeBlocks: if the console should be wrapped in code blocks and colored
```
Comment (granny) [inline src/main/resources/config/it.yml:192]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+# %username%:      il nome utente dell'utente che viene risposto a Discord
+#                   esempio: Notch
+# %message%:       il contenuto del messaggio che viene risposto a
+#
+# NOTE: La %reply% placeholderdeve essere presente nel formato se vuoi il DiscordToMinecraftMessageReplyFormat per visualizzare nel tuo messaggio.
+#
+DiscordToMinecraftChatMessageFormat: "[<aqua>Discord</aqua> | %toprolecolor%%toprolealias%<reset>] %name%%reply% » %message%"
+DiscordToMinecraftChatMessageFormatNoRole: "[<aqua>Discord</aqua>] %name%%reply% » %message%"
+DiscordToMinecraftAllRolesSeparator: " | "
+DiscordToMinecraftMessageReplyFormat: " (replying to %name%)"
```
Comment (granny) [inline src/main/resources/messages/it.yml:51]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 9]
```diff
@@ -0,0 +1,440 @@
...
+# Questo è il formato utilizzato quando si invia una linea dalla console al canale della console, se abilitato
+#
+# Placeholders Disponibili:
+# {level}:    livello di gravità del messaggio
+#              esempio: INFO, WARN, ERROR
+# {name}:     logger name
+#              example: Server
+# {datetime}: Data e ora corrente
+#              esempio: Sun Jan 1 15:30:45 PDT 2017
+# PlaceholderAPI placeholders are also supported
```
Comment (granny) [inline src/main/resources/messages/it.yml:131]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 10]
```diff
@@ -0,0 +1,440 @@
...
+DynmapChatFormat: "%message%"
+DynmapDiscordFormat: "[Dynmap] %name% » %message%"
+
+# Messaggio del canale della console del discord
+# Questo è il formato utilizzato quando si invia una linea dalla console al canale della console, se abilitato
+#
+# Placeholders Disponibili:
+# {level}:    livello di gravità del messaggio
+#              esempio: INFO, WARN, ERROR
+# {name}:     logger name
```
Comment (granny) [inline src/main/resources/messages/it.yml:127]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 11]
```diff
@@ -0,0 +1,440 @@
...
+#              example: Server
+# {datetime}: Data e ora corrente
+#              esempio: Sun Jan 1 15:30:45 PDT 2017
+# PlaceholderAPI placeholders are also supported
+#
+# DiscordConsoleChannelTimestampFormat: The date format that will be used for the {date} and {datetime} placeholders
```
Comment (granny) [inline src/main/resources/messages/it.yml:133]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 12]
```diff
@@ -0,0 +1,440 @@
...
+# DiscordChatChannelListCommandFormatOnlinePlayers: il messaggio all'inizio della lista, prima di tutti i nomi dei giocatori
+# DiscordChatChannelListCommandFormatNoOnlinePlayers: usato invece per quando non ci sono giocatori online
+# DiscordChatChannelListCommandPlayerFormat: il formato di come ogni giocatore dovrebbe apparire nell'elenco
+#   Available placeholders:
+#   %username%:     Nome utente del giocatore 
+#   %displayname%:  Visualizza nome da cose come soprannomi
+#   %primarygroup%: il nome del gruppo principale dell'utente
+#   %world%:        Il nome del giocatore del mondo è dentro
+#   %worldalias%:   Alias of World Player è in Via Multiverso-Core
+#   PlaceholderAPI placeholders are also supported
```
Comment (granny) [inline src/main/resources/messages/it.yml:166]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 13]
```diff
@@ -0,0 +1,440 @@
...
+# %freememory%:    Memoria gratuita del JVM in MB
+# %usedmemory%:    Memoria usata del JVM in MB
+# %totalmemory%:   Memoria totale del JVM in MB
+# %maxmemory%:     Memoria massima del JVM in MB
+# %freememorygb%:  Memoria gratuita del JVM in GB
+# %usedmemorygb%:  Memoria usata del JVM in GB
+# %totalmemorygb%: Memoria totale del JVM in GB
+# %maxmemorygb%:   Memoria massima del JVM in GB
+# %tps%:           TPS medio del server
+# PlaceholderAPI placeholders are also supported
```
Comment (granny) [inline src/main/resources/messages/it.yml:364]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 14]
```diff
@@ -0,0 +1,440 @@
...
+ChannelTopicUpdaterChatChannelTopicAtServerShutdownFormat: "Il server è offline da allora <t:%timestamp%:R> | %totalplayers% I giocatori unici sono mai entrati"
+ChannelTopicUpdaterConsoleChannelTopicAtServerShutdownFormat: "Il server è offline da allora <t:%timestamp%:R> | %serverversion%"
+
+# Discord Command Message.
+# Questo è il messaggio inviato ai giocatori quando corrono "/discord". Si consiglia di lasciare la sintassi del comando come parte di questo
+# Uso {INVITE} Come segnaposto per il collegamento di invito, le persone devono aderire al server Discord, usi DiscordInviteLink configurato in config.yml
+#
+DiscordCommandFormat: "&bUnisciti a noi sulla discordia a {INVITE}&b. Per il comando help., fare \"/discord ?\""
+
+# No permission message
```
Comment (granny) [inline src/main/resources/messages/it.yml:378]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 15]
```diff
@@ -0,0 +1,440 @@
...
+NoPermissionMessage: "&cNon hai il permesso di eseguire questo comando."
+
+# Unknown command message
```
Comment (granny) [inline src/main/resources/messages/it.yml:381]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 16]
```diff
@@ -0,0 +1,440 @@
...
+# %tps%:           TPS medio del server
+# PlaceholderAPI placeholders are also supported
+#
+ChannelTopicUpdaterChatChannelTopicFormat: "%playercount%/%playermax% GIOCATORI ONLINE. | %totalplayers% I giocatori unici sono mai entrati | Server ha iniziato <t:%starttimestamp%:R> | Ultimo aggiornamento <t:%timestamp%:R>"
+ChannelTopicUpdaterConsoleChannelTopicFormat: "TPS: %tps% | Mem: %usedmemorygb%GB usata/%freememorygb%GB libera/%maxmemorygb%GB max. | %serverversion%"
+# AtServerShutdownFormats ONLY supports %totalplayers%, %serverversion%, %timestamp%, & %date% / %time%
+ChannelTopicUpdaterChatChannelTopicAtServerShutdownFormat: "Il server è offline da allora <t:%timestamp%:R> | %totalplayers% I giocatori unici sono mai entrati"
+ChannelTopicUpdaterConsoleChannelTopicAtServerShutdownFormat: "Il server è offline da allora <t:%timestamp%:R> | %serverversion%"
+
+# Discord Command Message.
```
Comment (granny) [inline src/main/resources/messages/it.yml:372]: needs to be translated

File: src/main/resources/messages/it.yml [hunk 17]
```diff
@@ -0,0 +1,440 @@
...
+# %freememorygb%:  Memoria gratuita del JVM in GB
+# %usedmemorygb%:  Memoria usata del JVM in GB
+# %totalmemorygb%: Memoria totale del JVM in GB
+# %maxmemorygb%:   Memoria massima del JVM in GB
+# %tps%:           TPS medio del server
+# PlaceholderAPI placeholders are also supported
+#
+ChannelTopicUpdaterChatChannelTopicFormat: "%playercount%/%playermax% GIOCATORI ONLINE. | %totalplayers% I giocatori unici sono mai entrati | Server ha iniziato <t:%starttimestamp%:R> | Ultimo aggiornamento <t:%timestamp%:R>"
+ChannelTopicUpdaterConsoleChannelTopicFormat: "TPS: %tps% | Mem: %usedmemorygb%GB usata/%freememorygb%GB libera/%maxmemorygb%GB max. | %serverversion%"
+# AtServerShutdownFormats ONLY supports %totalplayers%, %serverversion%, %timestamp%, & %date% / %time%
```
Comment (granny) [inline src/main/resources/messages/it.yml:368]: needs to be translated

**Review by granny — CHANGES_REQUESTED** [review]
@Guglieee I apologize for the delay.

I've updated the configs and added english translations that were missing since new options have been added. As soon as these translations are done, I will merge this PR. 

Thank you for your patience.

File: src/main/resources/messages/it.yml [hunk 18]
```diff
@@ -0,0 +1,440 @@
...
+# Available placeholders Per giocatore / PlayerFirstJoin / PlayerLeave / PlayerDeath / PlayerAcheVement:
+# %displayname%:         Visualizza nome da cose come soprannomi
+# %username%:            Nome utente del giocatore 
+# %displaynamenoescapes%:         Visualizza il nome da cose come soprannomi senza sfuggire il formato di Discord (per l'uso in codice in linea e blocco del codice di codice)
+# %usernamenoescapes%:            Nome utente del lettore raw senza fuggire il formato di Discord (per l'uso in codice in linea e blocco del codice di codice)
+# %date%:                Data e ora corrente
+# %embedavatarurl%:      L'avatar dell'utente
+# %botavatarurl%:        L'avatar del bot
+# %botname%:             il nome del bot
+# PlaceholderAPI placeholders are also supported
```
Comment (ANutley) [inline src/main/resources/messages/it.yml:191]: Needs to be translated

File: src/main/resources/messages/it.yml [hunk 19]
```diff
@@ -0,0 +1,440 @@
...
+      Url: ""
+    Description: ""
+    Fields: []
+    ImageUrl: ""
+    Footer:
+      Text: ""
+      IconUrl: ""
+    Timestamp: false
+#
+# Available placeholders for PlayerLeave messages:
```
Comment (ANutley) [inline src/main/resources/messages/it.yml:251]: Needs to be translated

File: src/main/resources/messages/it.yml [hunk 20]
```diff
@@ -0,0 +1,440 @@
...
+      Url: ""
+    Description: ""
+    Fields: []
+    ImageUrl: ""
+    Footer:
+      Text: ""
+      IconUrl: ""
+    Timestamp: false
+#
+# Available placeholders for PlayerDeath messages:
```
Comment (ANutley) [inline src/main/resources/messages/it.yml:280]: Needs to be translated

File: src/main/resources/alerts/it.yml [hunk 21]
```diff
@@ -0,0 +1,122 @@
...
+#   #player - il giocatore per cui l'evento è per, se questo è un evento o un comando del giocatore
+#   #sender - Il mittente di comando, se questo è un avviso di comando
+#   #command - Il comando completo, senza barra leader, se è un avviso di comando
+#   #args - Gli argomenti di comando, se questo è un avviso di comando
+#   #allArgs - Gli argomenti di comando come una stringa, se questo è un avviso di comando
+#   #channel - Il canale di destinazione per questo avviso
+#   #jda - L'istanza JDA di DiscordSRV che usa per comunicare con Discord
+#
+# Sintassi / default:
+#  - Trigger: <event name or /command>
```
Comment (ANutley) [inline src/main/resources/alerts/it.yml:49]: Needs to be translated

File: src/main/resources/alerts/it.yml [hunk 22]
```diff
@@ -0,0 +1,122 @@
...
+#    Async: true # facoltativo, predefinito su true, non dovresti quasi mai cambiare questo
+#    Conditions:
+#      - property == < > <= >= value etc
+#    Channel: <discordsrv channel name>
+#    IgnoreCancelled: true # only for event alerts
```
Comment (ANutley) [inline src/main/resources/alerts/it.yml:54]: Needs to be translated

File: src/main/resources/alerts/it.yml [hunk 23]
```diff
@@ -0,0 +1,122 @@
...
+#      Author:
+#        ImageUrl: "{embedavatarurl}"
+#        Name: "{username} Ha fatto qualcosa"
+#        Url: ""
+#      ThumbnailUrl: ""
+#      Title:
+#        Text: ""
+#        Url: ""
+#      Description: ""
+#      Fields: [] # format is "title;value;inline" (eg. "Who joined?;%displayname%;true") or "blank" to add a blank field
```
Comment (ANutley) [inline src/main/resources/alerts/it.yml:72]: Needs to be translated

File: src/main/resources/config/it.yml [hunk 24]
```diff
@@ -0,0 +1,300 @@
...
+# Il canale della console è il canale di testo che riceve messaggi che vengono eseguiti come comandi del server
+# come una console, puoi scrivere su discord cio che vorresti scrivere nella console
+#
+# È possibile personalizzare il messaggio, inclusi la rimozione dei timestamps in message.yml
+#
+# DiscordConsoleChannelLogRefreshRateInSeconds: ogni quante righe nella console, le invia su discord
+# DiscordConsoleChannelUsageLog:
+#    %date%:  current date
+#              example: 2017-01-01
+#    PlaceholderAPI placeholders are supported
```
Comment (ANutley) [inline src/main/resources/config/it.yml:187]: Needs to be translated

File: src/main/resources/config/it.yml [hunk 25]
```diff
@@ -0,0 +1,300 @@
...
+# MinecraftDiscordAccountLinkedConsoleCommands: comandi da eseguire quando un account è collegato, vedere di seguito per il possibile placeholders
+# MinecraftDiscordAccountUnlinkedConsoleCommands: I comandi da eseguire quando è scollegato un account, vedere di seguito per i possibili segnaposti
+# %minecraftplayername%: player's Minecraft username
+#                         example: Notch
+# %minecraftuuid%:       player's uuid
+#                         example: Sai come appare un uuid
+# %discordid%:           linked discord account's id
+#                         example: 12345678901234567890
+# %discordname%:         linked discord account's username
+#                         example: Notch
```
Comment (ANutley) [inline src/main/resources/config/it.yml:278]: Needs to be fully translated

**Review by ANutley — CHANGES_REQUESTED** [review]
Again, apologies for the delay, but just a few more nitpicks before we can merge this

**Comment by granny** [comment]
Closing since there's been no activity for a while.

**Pull request closed without merging**